server/api/config.go

```diff
@@ -44,6 +46,29 @@ func newConfHandler(svr *server.Server, rd *render.Render) *confHandler {
 	}
 }
 
+// @Tags config
+// @Summary Update temporary configuration. Note that this is a temporary api and only support for lightning and BR. Don't call this API and it will be removed in release-5.0.
```

Comment:
do we need to add a warning in the return result?

Reply:
I think the `Warning` in @Description is enough.